sql/2021/privacy/nb_websites_with_gpc_well_known.sql

```diff
@@ -0,0 +1,22 @@
+#standardSQL
+# Pages that provide `/.well-known/gpc.json` for Global Privacy Control
+
+WITH pages_well_known AS (
+  SELECT
+    _TABLE_SUFFIX AS client,
+    JSON_VALUE(payload, "$._well-known") AS metrics
+  FROM
+    `httparchive.pages.2021_07_01_*`
+)
+
+SELECT
+  client,
+  COUNT(0) AS nb_websites -- crawled sites containing at leat one origin trial
+FROM
+  pages_well_known
+WHERE
+  JSON_VALUE(pages_well_known.metrics, '$."/.well-known/gpc.json".found') = "true"
+GROUP BY
+  client
+ORDER BY
+  client ASC
```

Comment:
Can this be simplified into one query?

Should it join with a totals query (so back up to two tables!) to give a percentage?

Reply:
Adjusted to a version with percentages.